src/mainwindow.cpp

```diff
@@ -602,6 +603,16 @@ void MainWindow::initFakeVim(QOwnNotesMarkdownTextEdit *noteTextEdit) {
 
     auto proxy = new FakeVimProxy(noteTextEdit, this, handler);
 
+    QSettings settings;
+    if (settings.value(QStringLiteral("Editor/useTabIndent")).toBool()) {
+        FakeVim::Internal::theFakeVimSettings()->item("et")->setValue(true);
```

Comment:
Can't you use `FakeVim::Internal::theFakeVimSettings()->item("et")->setValue(VARIABLE_HERE);` to get around code duplication?

Reply:
Wait, what are those two lines of `FakeVim::Internal::theFakeVimSettings()->item("et")->setValue(true);` doing? 😆

Reply:
Both times true?

Reply:
> Can't you use `FakeVim::Internal::theFakeVimSettings()->item("et")->setValue(VARIABLE_HERE);` to get around code duplication?

Yes, I guess

Reply:
> Wait, what are those two lines of `FakeVim::Internal::theFakeVimSettings()->item("et")->setValue(true);` doing? laughing

AH yes, one needs to be false.